#!/usr/bin/env python

PACKAGE="turn"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("use_personalized_angles", bool_t, 0, "Use personalized turning angles:", True)
gen.add("exit_turning_angle",            double_t, 0,  "Exit turning angle : ", 1.5,    0.0, 6.28319)
gen.add("entrance_turning_angle",            double_t, 0,  "Entrance turning angle : ", 1.5,    0.0, 6.28319)
gen.add("fixed_speed",            double_t, 0,  "Angular fixed speed : ", 0.5,    0.0, 0.52)

exit(gen.generate(PACKAGE, "turn_param", "TurnParam"))
